<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>661</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,0,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/images/repeat.png</normaloff>:/images/repeat.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_shuffle">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/images/shuffle.png</normaloff>:/images/shuffle.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>1.00</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/fastforward-icon-7.jpg</normaloff>:/images/fastforward-icon-7.jpg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.5</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/fastforward-icon-7.jpg</normaloff>:/images/fastforward-icon-7.jpg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.00</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/images/fastforward-icon-7.jpg</normaloff>:/images/fastforward-icon-7.jpg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_previous">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/images/music-player (1).png</normaloff>:/images/music-player (1).png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_stop">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/images/stop-button.png</normaloff>:/images/stop-button.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_play">
               <property name="minimumSize">
                <size>
                 <width>82</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">min-width: 80px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/images/play-button.png</normaloff>:/images/play-button.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_pause">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_next">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/images/music-player.png</normaloff>:/images/music-player.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnMute">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="Resources.qrc">
                 <normaloff>:/images/Voice_48px.png</normaloff>:/images/Voice_48px.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sliderVolume">
               <property name="value">
                <number>99</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Playlist">
    <property name="title">
     <string>Add Playlist</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd_Media"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
   <addaction name="menuAdd_Playlist"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionactionAddPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/play-button.png</normaloff>:/images/play-button.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/stop-button.png</normaloff>:/images/stop-button.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionAdd_Media">
   <property name="text">
    <string>Add Media</string>
   </property>
  </action>
  <action name="actionactionAddPlaylist">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/playlist.png</normaloff>:/images/playlist.png</iconset>
   </property>
   <property name="text">
    <string>actionAddPlaylist</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
